examples/kubernetes-custom-vnet/kubernetes-custom-vnet-maxpods.json

```diff
@@ -0,0 +1,60 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "networkPolicy": "azure",
+        "maxPods": 30

```

Comment:
Is this number going to be used at all or is it overridden everywhere?

Reply:
This number will be honored if no `--max-pods` is set in KubernetesConfig.KubeletConfig. This is backward compatibility case talked in the previous comment.

Reply:
I mean in this example file, since --max-pods is set for both agent and master kubelet. Won't it be confusing for users if we include a value in an example file that is ignored?

Reply:
Actually no master KubernetesConfig.KubeletConfig is set in this example. The above value will be used for max pods at master nodes. The other two values are used for agentpool1 and nodepool2.

Reply:
My bad, I misread, I saw two values and assumed it was master and agent. Looks good!